Services/Interfaces/web_interface/advanced_controllers/tentacles.py

```diff
@@ -18,14 +18,33 @@
 
 import tentacles.Services.Interfaces.web_interface.advanced_controllers as advanced_controllers
 import tentacles.Services.Interfaces.web_interface.login as login
+from tentacles.Services.Interfaces.octo_ui2.models.octo_ui2 import import_cross_origin_if_enabled, dev_mode_is_on
 import tentacles.Services.Interfaces.web_interface.util as util
 import tentacles.Services.Interfaces.web_interface.models as models
 import octobot_commons.authentication as authentication
 
-
-@advanced_controllers.advanced.route("/tentacles")
-@login.active_login_required
-def tentacles():
+tentacles_route = "/tentacles"
+if cross_origin := import_cross_origin_if_enabled():
+    if dev_mode_is_on:
+        @advanced_controllers.advanced.route(tentacles_route)
+        @login.login_required_when_activated
+        @cross_origin(origins="*")
+        def tentacles():
+            return _tentacles()
+    else:
+        @advanced_controllers.advanced.route(tentacles_route)
+        @login.active_login_required
+        @cross_origin(origins="*")
```

Comment:
Could you add a variable for the origin value please? So we can change it easily if needed